modules/compliance-operator-rosa-installation.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-management/compliance-operator-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-compliance-operator-rosa_{context}"]
+= Installing the Compliance Operator on ROSA hosted control planes (HCP)
+
+As of the Compliance Operator 1.5.0 release, the operator is tested against Red Hat
+OpenShift Service on AWS (ROSA) using hosted control plane topologies (HCP).
+
+ROSA HCP clusters have restricted access to the control plane, which is managed
+by Red Hat. By default, the Compliance Operator will schedule to nodes within
+the `master` node pool, which isn't available in ROSA HCP installations. This
+requires you to configure the `Subscription` object in a way that allows the
+operator to schedule on available node pools. This step is necessary for a
+successful installation on ROSA HCP clusters.
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+.Procedure
+
+. Define a `Namespace` object:
++
+.Example `namespace-object.yaml`
```

Comment:
```suggestion
.Example `namespace-object.yaml` file
```
I won't comment on all of these, but apply consistently like this for any similar block titles.